packages/subgraph/tests/addresslist-voting/addresslist-voting.test.ts

```diff
@@ -111,7 +112,7 @@ test('Run AddresslistVoting (handleProposalCreated) mappings with mock event', (
   let entityID =
     Address.fromString(CONTRACT_ADDRESS).toHexString() +
     '_' +
-    BigInt.fromString(proposalId).toHexString();
+    bigIntToBytes32(BigInt.fromString(proposalId));
```

Comment:
thats the beauty now, in tests you should be using `getProposalId`.

Reply:
outdated, but switched to import `PROPOSAL_ENTITY_ID`